README.md

```diff
@@ -1,2 +1,10 @@
 # EmbeddedChat
 An easy to use full-stack component  (ReactJS + backend behaviors)  embedding Rocket.Chat into your webapp
+
+## Development
```

Comment:
Please describe what this is.    Show a screen capture of how it looks like.

How will  the potential contributor be able to test his code?

How can this component be embedded into a web page or react app as it exists right now?